.github/workflow/BuildAndTest.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on:
+  push:
+    branches: [dev,master]
```

Comment:
I suggest to limit the action to dev branch because no PR is accepted on the master branch